_posts/2021-11-22-6.0.1-release.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "Apache Arrow 6.0.1 Release"
+date: "2021-11-22 00:00:00 -0600"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 6.0.1 release.
+This is mostly a bugfix release that includes [**30 resolved issues**][1]
+from [**16 distinct contributors**][2]. See the Install Page to learn how to
+get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 6.0.0 release, Joris Van den Bossche has joined the Project Management Committee (PMC). 
+Thanks for your contributions and participation in the project!
+
+## Documentation
+
+A version switcher is now available in the Python and C++ documentation
+to access different versions of the documentation.
+
+## C++ notes
+
+* Trying to join with a list column will now report an unsupported operation
+  error instead of crashing.
+* Dictionaries are now supported as an input in hash joins
+* Fixed a potential data loss in S3 multipart upload
+
+## Python notes
+
+* Dataset api now supports `existing_data_behavior` option when writing datasets.
+* Installing pyarrow from source distribution now works with setuptools 58.5
+
+## R notes
+
+* Fixed a crash when summarizing after filtering to no rows
+* Added bindings for `str_count`
+
+For more details, see the [complete R changelog][4]. 
+
+## Go notes
+
+* Arrow and Parquet modules are properly brought together under a top level `github.com/apache/arrow/go` module which can be installed as `github.com/apache/arrow/go/v6/arrow@v6.0.1`.
+
```

Comment:
```suggestion
## Ruby notes

Thanks to [simpl1g](https://github.com/simpl1g) Ruby now supports explicit initialization of S3 APIs. An introduction
to using Red Arrow is available in the [main repository](https://github.com/apache/arrow/blob/master/ruby/README.md), look for more updates in upcoming releases.
```

Reply:
6.0.1 doesn't include the feature.
7.0.0 will ship it.

Reply:
Ok. Maybe still helpful to highlight the introduction to using Ruby? This is not included in the release, but it is very helpful for anyone looking to get started with the Ruby bindings.

Reply:
6.0.1 is a patch release.
We should not do it for a patch release. Release annonces for patch releases should focus on only bug fixes.

Reply:
Ok, thanks.